### PR TITLE
<kbd> tag added

### PR DESCRIPTION
**Pull request by varghesejose2020**

### CODE REVIEW

**Comment by neflyte** [comment]
@varghesejose2020 Thanks for your contribution! Please update your branch. :)

**Review by neflyte — CHANGES_REQUESTED** [review]
Please update your branch to pick up the latest changes.

**Comment by varghesejose2020** [comment]
> Please update your branch to pick up the latest changes.

Done

**Comment by varghesejose2020** [comment]
Seperate PR..

**Review by neflyte — APPROVED** [review]

**Comment by cwarnermm** [comment]
@varghesejose2020 - Including all keyboard shortcut formatting changes in a single PR helps simplify and streamline the internal review process, and results in your PR being merged more quickly into the code base.

**Comment by varghesejose2020** [comment]
> @varghesejose2020 - Including all keyboard shortcut formatting changes in a single PR helps simplify and streamline the internal review process, and results in your PR being merged more quickly into the code base.

ok

**Comment by varghesejose2020** [comment]
Hi I added founded keyboard shortcut formatting changes in developer documents. If I missed any thing please update me @cwarnermm

File: site/content/contribute/more-info/webapp/developer-setup.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ Set up your development environment for building, running, and testing the Matte
 
     Refreshing http://localhost:8065 should now load the UI.
 
-    If you would like the webapp to automatically refresh as you edit the source code you can install and enable the live reload script injection extension for your web browser ([Chrome](https://chrome.google.com/webstore/detail/remotelivereload/jlppknnillhjgiengoigajegdpieppei/related?hl=en) | [Firefox](https://addons.mozilla.org/en-US/firefox/addon/livereload-web-extension/)). Then, before running webpack run:
+    If you would like the webapp to automatically refresh as you edit the source code you can install and enable the live reload script injection extension for your web browser [![Chrome](https://img.shields.io/badge/Chrome-navy?\&style=plastic\&logo=googlechrome\&logoColor=4285F4)](https://chrome.google.com/webstore/detail/remotelivereload/jlppknnillhjgiengoigajegdpieppei/related?hl=en) | [![Firefox](https://img.shields.io/badge/Firefox-navy?\&style=plastic\&logo=firefoxbrowser\&logoColor=FF7139)](https://addons.mozilla.org/en-US/firefox/addon/livereload-web-extension/). Then, before running webpack run:
```
Comment (neflyte) [inline site/content/contribute/more-info/webapp/developer-setup.md:72]: While there is only one link to the Mozilla Addons site, there are several that link to the Chrome webstore. I think we should tackle these in a separate ticket so they are used consistently. Please feel free to create a new PR if you would like to update the other links with images as well. :slightly_smiling_face: 

For this PR, please revert the changes to these links.

**Review by neflyte — CHANGES_REQUESTED** [review]

File: site/content/contribute/more-info/webapp/developer-setup.md (hunk 1 above)
Comment (varghesejose2020) [reply site/content/contribute/more-info/webapp/developer-setup.md]: reverted

**Review by cwarnermm — APPROVED** [review]
Thank you, @varghesejose2020!

**Review by neflyte — APPROVED** [review]

**Comment by varghesejose2020** [comment]
@cwarnermm

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**